alembic/versions/eaca44fbeabe_add_soft_delete_to_events.py

```diff
@@ -0,0 +1,28 @@
+"""add soft delete to events
+
+Revision ID: eaca44fbeabe
+Revises: 404358628973
+Create Date: 2019-02-06 07:00:32.174447
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eaca44fbeabe'
+down_revision = '404358628973'
```

Comment:
Please rebase and point this to the new alembic file that has been merged

Reply:
Okay @RonKbS on it

Reply:
Sure it is currently what I am working on. @RonKbS